frontend-opal/components/Header/styles.js

```diff
@@ -0,0 +1,88 @@
+import { StyleSheet } from 'react-native';
+
+let displayBorder = (width = 2, style = "solid", color = "black") => (
+    {
+        borderWidth: width,
+        borderStyle: style,
+        borderColor: color,
+    }
+)
+
+export default StyleSheet.create({
+/* ------------------------ Search Area ------------------------ 
+    - a list icon, 
+    - an search input
+    - a search button
```

Comment:
@Yuan-EPAM this doesn't add any more information than the names of the css classes, which is good, so you can get rid of this comment